### ISSUE TITLE
[TextAnalytics] Address feedback from Extractive Summarization board review

### ISSUE THREAD

**Issue by kinelski** [Azure/azure-sdk-for-net#23297]
Link to the board review issue: https://github.com/Azure/azure-sdk/issues/3167

**Comment by maririos** [comment]
@kinelski could you list in here the specific items that .NET needs to address after the arch board?

**Comment by maririos** [comment]
List is:
- Rename classifyDocumentSingleCategory to singleCategoryClassify
- Rename classifyDocumentMultipleCategories to multipleCategoryClassify
- Rename DocumentClassification to ClassificationCategory

**Comment by maririos** [comment]
@AhmedLeithy or @ZulaMostafa could either of you verify that we applied these renames in the SDK?

**Comment by AhmedLeithy** [comment]
> @AhmedLeithy or @ZulaMostafa could either of you verify that we applied these renames in the SDK?

Yes, all three of these renames were done in my PRs. There was also the following property renames that Krista pointed out:

`SingleCategoryClassifyResult.ClassificationCategory` -> `SingleCategoryClassifyResult.Classification`
`MultiCategoryClassifyResult.ClassificationCategories` -> `MultiCategoryClassifyResult.Classifications`

**Comment by maririos** [comment]
So we can close this issue then. right?
